### DISCUSSION TITLE
Display the star rating for recommended difficulty on beatmap listing

### DISCUSSION THREAD

**Discussion by felipemarins** [ppy/osu#30394] [Ideas] [answered]
I think it would be cool if the star rating was shown next to the "recommended difficulty" filter like it is on the website.

To demonstrate visually, this is what I'm talking about:
![image](https://github.com/user-attachments/assets/c15a1891-4b93-41b8-8357-9984e943b78b)

**Comment by peppy** [comment]
Definitely something we should have. Implementing is a bit complicated due to how things are currently structured.

**Comment by Joehuu** [comment, marked as answer]
https://github.com/ppy/osu/pull/31101
